user_guide_src/source/concepts/security.rst

```diff
@@ -578,8 +573,361 @@ Do not mitigate SSRF via the use of a deny list or regular expression. Attackers
 have payload lists, tools, and skills to bypass deny lists.
 
 CodeIgniter provisions
-======================
+----------------------
+
+- :doc:`../libraries/validation` library
+- :doc:`HTTP library <../incoming/incomingrequest>` provides for
+  :ref:`input field filtering <incomingrequest-filtering-input-data>`
+
+******************************
+OWASP API Security Top 10 2023
+******************************
+
+API1:2023 Broken Object Level Authorization
+===========================================
+
+APIs tend to expose endpoints that handle object identifiers, creating a wide
+attack surface of Object Level Access Control issues. Object level authorization
+checks should be considered in every function that accesses a data source using
+an ID from the user.
+
+OWASP recommendations
+---------------------
+
+- Implement a proper authorization mechanism that relies on the user policies and
```

Comment:
In this case a good job is done by **lonnieezell**:
https://github.com/lonnieezell/forum-example/blob/develop/_docs/policies.md